#SAK-14298 Minor Change checkbox is ignored 

=svnm= https://source.sakaiproject.org/svn/rwiki/trunk@51470 rwiki

#Fill in the SAK or something unique, leave it like this if theres no real patch
--- /dev/null
+++ SAK-14298.tmp
@@ -0,0 +1 @@
+ 
